static/app/components/acl/feature.tsx

```diff
@@ -27,7 +29,7 @@ type Props = {
    *
    * Use `organizations:` or `projects:` prefix strings to specify a feature with context.
    */
-  features: string[];
+  features: string | string[];
```

Comment:
Are there any scenarios where we're using multiple features?

Reply:
Presumably yes, right? `requireAll` is a prop too. The behaviour for the array is weird since you might want `&&` or `||` for features, I think the default is or, which might be unexpected.

Reply:
Without looking at it, I would assume we `&&` in case of multiple features, but it is always tough to express that through a single interface (and we might not even need to). If we do need to though, I think the most expressive without making it unnecessarily complex would be to create separate signatures like `{feature: string} | {eitherOf: string[]} | {oneOf: string[]}`. 

I will check tomorrow to see if we can drop the string[] signature and just simplify this :)

Reply:
Checked and we do have some multi-feature use so we cant drop it